aiohttp/helpers.py

```diff
@@ -111,12 +112,40 @@ def strip_auth_from_url(url):
         return url.with_user(None), auth
 
 
+def netrc_from_env():
+    netrc_obj = None
+    netrc_path = os.environ.get('NETRC')
+    try:
+        if netrc_path is not None:
+            netrc_path = Path(netrc_path)
+        else:
+            home_dir = Path.home()
+            if os.name == 'nt':  # pragma: no cover
+                netrc_path = home_dir.joinpath('_netrc')
+            else:
+                netrc_path = home_dir.joinpath('.netrc')
+
+        if netrc_path and netrc_path.is_file():
+            try:
+                netrc_obj = netrc.netrc(str(netrc_path))
+            except (netrc.NetrcParseError, IOError) as e:
```

Comment:
Rename `IOError` to `OSError` -- they are aliases but `OSError` is the preferable name.